python/cugraph/tests/test_nx_convert.py

```diff
@@ -29,19 +29,66 @@
     import networkx as nx
 
 
+def _compare_graphs(nxG, cuG, has_wt=True):
+    assert nxG.number_of_nodes() == cuG.number_of_nodes()
+    assert nxG.number_of_edges() == cuG.number_of_edges()
+
+    cu_df = cuG.view_edge_list().to_pandas()
+    if has_wt is True:
+        cu_df = cu_df.drop(columns=["weights"])
+    cu_df = cu_df.sort_values(by=["src", "dst"]).reset_index(drop=True)
+
+    nx_df = nx.to_pandas_edgelist(nxG)
+    if has_wt is True:
+        nx_df = nx_df.drop(columns=["weight"])
+    nx_df = nx_df.rename(columns={"source": "src", "target": "dst"})
+    nx_df = nx_df.astype('int32')
+    nx_df = nx_df.sort_values(by=["src", "dst"]).reset_index(drop=True)
+
+    assert cu_df.to_dict() == nx_df.to_dict()
+
+
+@pytest.mark.parametrize("graph_file", utils.DATASETS)
+def test_networkx_compatibility(graph_file):
```

Comment:
optional: maybe rename to "test_networkx_graph_compatibility" since just "compatibility" is pretty broad.

Reply:
On second though, the test is scoped to the "nx_convert" test file, so "convert" isn't as broad of a term in that sense. Your call.